js/gallerybutton.js

```diff
@@ -61,9 +61,7 @@ $(document).ready(function () {
 			GalleryButton.button =
 				$('<div id="gallery-button" class="button view-switcher">' +
 						'<div id="button-loading"></div>' +
-					'<img class="svg" src="' + OC.imagePath('core', 'actions/toggle-pictures.svg') +
-					'"' +
-					'alt="' + t('gallery', 'Gallery view') + '"/>' +
+					'<span class="icon-toggle-pictures"></span>' +
```

Comment:
Just tested on Files and seems all of these are gone anyway, so if @jancborchardt is OK with this, it can be merged.